exonum-java-binding/core/src/main/java/com/exonum/binding/core/runtime/ServiceRuntime.java

```diff
@@ -400,7 +400,10 @@ private void logApiMountEvent(ServiceWrapper service, String serviceApiPath, Rou
         );
   }
 
-  private ServiceWrapper getServiceById(Integer serviceId) {
+  /**
+   * Finds service instance by its id.
```

Comment:
There is a difference between get and find, not reflected in this Javadoc: this will throw if it is not found, i.e., the client is assumed to pass a valid id.